Gems/MotionMatching/Code/Source/CsvSerializers.cpp

```diff
@@ -109,27 +109,35 @@ namespace EMotionFX::MotionMatching
         WriteLine(m_tempBuffer);
     }
 
-    void PoseWriterCsv::WritePose(Pose& pose)
+    void PoseWriterCsv::WritePose(Pose& pose, const ETransformSpace transformSpace)
     {
         if (!IsReady() || !m_actorInstance)
         {
             return;
         }
 
-        SavePoseToString(pose, m_tempBuffer);
+        SavePoseToString(pose, transformSpace, m_tempBuffer);
         WriteLine(m_tempBuffer);
     }
 
-    void PoseWriterCsv::SavePoseToString(Pose& pose, AZStd::string& outText)
+    void PoseWriterCsv::SavePoseToString(Pose& pose, const ETransformSpace transformSpace, AZStd::string& outText)
     {
         const size_t numEnabledJoints = m_actorInstance->GetNumEnabledNodes();
         outText.clear();
         outText.reserve(10 * 3 * 3 * numEnabledJoints);
 
+        Transform transform;
```

Comment:
suggestion: Can we just move this to first use inside the loop, having this hang around at the outer scope likely isn't worth it

Reply:
Good point, ancient optimization removed ;). Will come with the next PR.